doc/dev/architecture.md

```diff
@@ -175,43 +175,69 @@ internally.
 ## Mount Point Configuration
 
 `kdb mount` creates a **mount point configuration** as shown
-in the example below. `fstab` is a unique name
+in the example below. `/hosts` is a unique name
 within the mount point configuration provided by the administrator.
+To escape the `/` character, the name is changed to `\/hosts` in the
+configuration.
 
 Example for a mount point configuration:
 
 ```
-system:/elektra/mountpoints
-system:/elektra/mountpoints/fstab
-system:/elektra/mountpoints/fstab/config
-system:/elektra/mountpoints/fstab/config/path=fstab
-system:/elektra/mountpoints/fstab/config/struct=list FStab
-system:/elektra/mountpoints/fstab/config/struct/FStab
-system:/elektra/mountpoints/fstab/config/struct/FStab/device
-system:/elektra/mountpoints/fstab/config/struct/FStab/dumpfreq
-system:/elektra/mountpoints/fstab/config/struct/FStab/mpoint
-system:/elektra/mountpoints/fstab/config/struct/FStab/options
-system:/elektra/mountpoints/fstab/config/struct/FStab/passno
-system:/elektra/mountpoints/fstab/config/struct/FStab/type
-system:/elektra/mountpoints/fstab/errorplugins
-system:/elektra/mountpoints/fstab/errorplugins/#5#resolver#resolver#
-system:/elektra/mountpoints/fstab/getplugins
-system:/elektra/mountpoints/fstab/getplugins/#0#resolver
-system:/elektra/mountpoints/fstab/getplugins/#5#fstab#fstab#
-system:/elektra/mountpoints/fstab/mountpoint /fstab
-system:/elektra/mountpoints/fstab/setplugins
-system:/elektra/mountpoints/fstab/setplugins/#0#resolver
-system:/elektra/mountpoints/fstab/setplugins/#1#struct#struct#
-system:/elektra/mountpoints/fstab/setplugins/#2#type#type#
-system:/elektra/mountpoints/fstab/setplugins/#3#path#path#
-system:/elektra/mountpoints/fstab/setplugins/#3#path#path#/config
-system:/elektra/mountpoints/fstab/setplugins/#3#path#path#/config/path/allow=proc tmpfs none
-system:/elektra/mountpoints/fstab/setplugins/#5#fstab
-system:/elektra/mountpoints/fstab/setplugins/#7#resolver
+system:/elektra/mountpoints/\/hosts
+system:/elektra/mountpoints/\/hosts/config
+system:/elektra/mountpoints/\/hosts/config/glob/set/#0
+system:/elektra/mountpoints/\/hosts/config/glob/set/#1
+system:/elektra/mountpoints/\/hosts/config/glob/set/#2
+system:/elektra/mountpoints/\/hosts/config/glob/set/#3
+system:/elektra/mountpoints/\/hosts/config/glob/set/#4
+system:/elektra/mountpoints/\/hosts/config/glob/set/#4/flags
+system:/elektra/mountpoints/\/hosts/config/mountpoint
+system:/elektra/mountpoints/\/hosts/config/path
+system:/elektra/mountpoints/\/hosts/error
+system:/elektra/mountpoints/\/hosts/error/rollback
+system:/elektra/mountpoints/\/hosts/error/rollback/#0
+system:/elektra/mountpoints/\/hosts/error/rollback/#0/label (="resolver")
```

Comment:
I like it this way. I assume you considered the possibility that all plugins are first defined and within the positioning only their label is used? (Separation of instantiation and using them at specific positions)

Reply:
Again not a file I wrote, and actually out-of-date